README.adoc

```diff
@@ -351,23 +351,25 @@ Where :
 - SHA1: is the given git SHA1 of the james-project repository to merge.
 - RESULTING_BRANCH: is the branch created when merging.
 
-=== Generating a debian package for James
+=== Generating a package for James
 
-You can generate a debian package for James by using the following process.
+You can generate a Debian package or an RPM package for James by using the following process.
```

Comment:
s#RPM#REHL/CentOS#

Reply:
do you think it won't work on Suse? :)

Reply:
or Mandriva :trollface:

Reply:
Either you write "deb / rpm" or distribution names, but the mix is unfortunate

Reply:
changed